linera-core/src/client/mod.rs

```diff
@@ -1750,9 +1750,12 @@ where
         #[cfg(with_metrics)]
         let _latency = metrics::SYNCHRONIZE_CHAIN_STATE_LATENCY.measure_latency();
 
-        let (epoch, mut committees) = self.epoch_and_committees(chain_id).await?;
+        let (epoch, mut committees) = match self.epoch_and_committees(chain_id).await? {
+            (Some(epoch), committees) => (epoch, committees),
+            (None, _) => self.known_committees().await?, // Chain is not active yet.
```

Comment:
This is a bit confusing for a number of reasons:
- both `epoch_and_committees` and `known_committees` return the same types
- `known_commitees` calls `epoch_commitees(chain_id)` as well

IIUC, in the case of "chain not active yet" we want to fetch the known committee from the admin chain, is that right?

Reply:
Yes. TBH I'm not sure what the "right committee" is in all those cases where we just need _nodes_ to synchronize from, as opposed to actual validators to ask to sign something. But maybe that's beyond the scope of this PR, and something we should just fix on `main` instead?

Reply:
OK, agreed. BUT an empty committee can't be a right committee ever, right?

Reply:
That's true, yes.